automation/roles/patroni/tasks/main.yml

```
@@ -942,8 +939,12 @@
         port: "{{ postgresql_port }}"
       register: postgresql_user_result
       loop:
-        - { name: '{{ patroni_superuser_username }}', password: '{{ patroni_superuser_password }}', flags: 'SUPERUSER' }
-        - { name: '{{ patroni_replication_username }}', password: '{{ patroni_replication_password }}', flags: 'LOGIN,REPLICATION' }
+        - { name: "{{ patroni_superuser_username }}", password: "{{ patroni_superuser_password }}", flags: "SUPERUSER" }
+        - {
+            name: "{{ patroni_replication_username }}",
+            password: "{{ patroni_replication_password }}",
+            flags: "LOGIN,REPLICATION",
+          }
```

```suggestion
        - { name: "{{ patroni_replication_username }}", password: "{{ patroni_replication_password }}", flags: "LOGIN,REPLICATION" }
```

I don't see the point in such formatting for just one line.

---

This is what prettier does. There are built-in rules for when there is a block-like style and for when not.

There is not really a way to have a custom view on these, either going with prettier or leaving it out entirely (your choice).

---

I get it, thanks for clarifying.